extensions/wikia/AchievementsII/services/AchAwardingService.class.php

```diff
@@ -423,7 +424,7 @@ private function processInTrackPicture() {
 							'list' => 'imageusage',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "imageusage" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)